src/components/home/home.jsx

```diff
@@ -80,6 +80,14 @@ class HomePage extends React.Component {
               </ul>
         </section>
 
+        <section id="home-volunteer">
+            <h1>Want to join the team?</h1>
+            <p>We welcome volunteers!</p>
+            <p>We work remotely and asynchronously using online collaboration tools like GitHub.</p>
```

Comment:
lessen the space between the last two paragraphs.

Reply:
@ambergkim Are you okay with a <br />, or would you prefer a <p> with reduced margins?

Reply:
@therealslimgrady The less clutter on the DOM the better. So the simpler the html the better.